domain/model/state/modelstate_test.go

```diff
@@ -267,3 +267,29 @@ func (s *modelSuite) TestGetModelMetricsNotFound(c *gc.C) {
 	_, err := state.GetModelMetrics(context.Background())
 	c.Assert(err, jc.ErrorIs, modelerrors.NotFound)
 }
+
+func (s *modelSuite) TestGetModelCloudType(c *gc.C) {
```

Comment:
Thanks @CodingCookieRookie, can you also please write a test to assert the not found error? There should be examples in this file already.